.github/ISSUE_TEMPLATE/validation-rule-request.md

```diff
@@ -0,0 +1,47 @@
+---
+name        : Validation rule request
+about       : A detailed specification for a new SARIF validation rule to add to the Sarif.Multitool validate command.
+title       : "[RULE REQUEST] Concise description of new analysis"
+labels      : validation-rule-request
+assignees   : ''
+
+---
+
+***********************************************************************************************************
+
+# Rule Proposal: [Friendly Rule Name]
+- *Synopsis*                : [Brief summary of the rule, include code snippets if possible.]
+
+- *[Violation Example]*     : [Optional, Include code snippet which should trigger the violation.]
+
+- *[No Violation Example]*  : [Optional, Include code snippet which demostrates ideal condition (no violation).]
+
+***********************************************************************************************************
+
+### Rule metadata
+- [*Id*]                : [Should be formatted as `SARIF1nnn`, leave blank if unsure]
+- *Name*                : [Provide a friendly symbolic name for the rule in PascalCase.]
+- *Level*               : [Possible values are: `error`, `warning`, or `note`.]
+- *Short description*   : [Short description of the rule.]
+- *Full description*    : [Full description, should describe usage of the rule and any other relevant information.]
+
+- User-facing strings:
+  Each rule has one or more result message strings, each with symbolic name in PascalCase.
```

Comment:
>E [](start = 2, length = 1)

As authored, this will appear on the same line as "User-facing strings". Is that what you intended? If so, did you mean to enclose it in square brackets as you did for the other field descriptions?

Reply:
oh, yes - this is supposed to be an instruction! thanks, will fix.

---
In reply to: [435556107](https://github.com/microsoft/sarif-sdk/pull/1903#discussion_r435556107) [](ancestors = 435556107)